src/component/TooltipBoundingBox.tsx

```diff
@@ -138,18 +112,21 @@ export class TooltipBoundingBox extends PureComponent<TooltipBoundingBoxProps, S
     };
 
     return (
-      // This element allow listening to the `Escape` key.
-      // See https://github.com/recharts/recharts/pull/2925
-      <div
-        tabIndex={-1}
-        className={cssClasses}
-        style={outerStyle}
-        ref={node => {
-          this.wrapperNode = node;
-        }}
-      >
-        {children}
-      </div>
+      <g>
+        <foreignObject x="0" y="0" width="100%" height="100%" style={{ pointerEvents: 'none' }}>
+          {/* This element allow listening to the `Escape` key. // See https://github.com/recharts/recharts/pull/2925 */}
+          <div
+            // @ts-expect-error typescript library does not recognize xmlns attribute
+            xmlns="http://www.w3.org/1999/xhtml"
```

Comment:
Why is this necessary?

Reply:
I added a comment in an attempt to explain